build/openresty/BUILD.openresty.bazel

```diff
@@ -194,6 +231,46 @@ CONFIGURE_OPTIONS = [
         "--with-ld-opt=\"-L$$EXT_BUILD_DEPS$$/lib -Wl,-Bsymbolic-functions -Wl,-z,relro\"",
     ],
     "//conditions:default": [],
+}) + select({
+    "@platforms//os:linux": [
+        # neded for wasmx module
+        # although this is centos7 specific, the flag will work on any GNU linker
+        # we place it here to skip macos, which uses darwin ld
+        # https://github.com/Kong/ngx_wasm_module/commit/e70a19f53e1dda99d016c5cfa393652720959afd
+        "--with-ld-opt=\"-Wl,--allow-multiple-definition\"",
```

Comment:
I am not sure it is a good idea to unconditionally add this flag... Do we maintain centos7 packages for Kong Gateway?